src/Http/Http.Abstractions/src/PublicAPI.Shipped.txt

```diff
@@ -149,7 +143,6 @@ Microsoft.AspNetCore.Http.QueryString.Value.get -> string?
 Microsoft.AspNetCore.Http.RequestDelegate
 Microsoft.AspNetCore.Http.RequestTrailerExtensions
 Microsoft.AspNetCore.Http.ResponseTrailerExtensions
-Microsoft.AspNetCore.Http.StatusCodes
```

Comment:
We [can't edit PublicAPI.Shipped.txt](https://dev.azure.com/dnceng/public/_build/results?buildId=1265318&view=logs&jobId=efda9e5f-cf98-536e-4181-9b6c13ac35b3&j=efda9e5f-cf98-536e-4181-9b6c13ac35b3&t=de5280bb-c511-517b-1959-798dcf9a84da) as that indicates a breaking change. If we do this, we would need to [type forward](https://github.com/dotnet/aspnetcore/blob/8b30d862de6c9146f466061d51aa3f1414ee2337/src/Http/Http.Features/src/Properties/AssemblyInfo.cs#L7-L10).

That said I think the better approach is to move IBadRequestExceptionFeature to Http.Abstractions but keep the namespace like IRouteValuesFeature.

Or we can just make `IBadRequestExceptionFeature.Error` a plain-old `Exception` and let the listener cast to BadHttpRequestException if they care that much about the StatusCode which is the only additional property.

Reply:
Thanks, I just pushed an update that follows the `IRouteValuesFeature` approach.